assets/js/components/DataBlockGroup.js

```diff
@@ -96,11 +101,22 @@ export default function DataBlockGroup( { className, children } ) {
 		} );
 	};
 
+	const resetFontSizes = ( blocks ) => {
+		blocks.forEach( ( block ) => {
+			const dataPoint = block?.querySelector(
+				'.googlesitekit-data-block__datapoint'
+			);
+			if ( dataPoint ) {
+				dataPoint.style.fontSize = '';
+			}
+		} );
+	};
```

Comment:
Hmm, thanks for updating this @ankitrox, however - it seems that if we don't need this function to be async, we can revert it entirely and use `setFontSizes( blocks, '' )` instead, as we do on `develop`.

Reply:
Thanks @techanvil , I have made that change in [de5dbd7](https://github.com/google/site-kit-wp/pull/10899/commits/de5dbd7cb2177b069f351c5929e5a427a938e3e5) and assigning it to you.